### DISCUSSION TITLE
Localisation

### DISCUSSION THREAD

**Discussion by SchraderR** [SchraderR/sp-tarkov-client#167] [General]
Localization

Tool: https://crowdin.com/
GitHub-Integration: https://support.crowdin.com/github-integration/

Initial Language: English
Next Languages: [German, Korean]

**Comment by TPL2007** [comment]
I'm Hanaku1999 hi

**Reply by SchraderR** [reply]
Hey. Thanks for your request. I will look into the tools and get back to you. 

As far I know, you will get a list with strings which need to be translated into your language (thanks a lot!)
I will take this list and add it technically in the app and later you can switch the language.
